src/fable-library/Util.ts

```diff
@@ -671,25 +671,25 @@ export function partialApply(arity: number, f: Function, args: any[]): any {
         // Wrap arguments to make sure .concat doesn't destruct arrays. Example
         // [1,2].concat([3,4],5)   --> [1,2,3,4,5]    // fails
         // [1,2].concat([[3,4],5]) --> [1,2,[3,4],5]  // ok
-        return (a1: any) => f.apply(null, args.concat([a1]));
+        return (a1: any) => f.apply(undefined, args.concat([a1]));
```

Comment:
Just aesthetics, but for `apply` it seems either `null` or `undefined` will work and the examples in MDN use `null`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply

Reply:
@alfonsogarciacaro Yes, the second note in the MDN article says `When the first argument is undefined or null ...`